src/common/files.go

```diff
@@ -0,0 +1,128 @@
+package common
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+// IsExecOwner checks if specified file has owner execute permissions
+func IsExecOwner(path string) (bool, error) {
+	fileInfo, err := os.Stat(path)
+	if err != nil {
+		return false, err
+	}
+
+	perm := fileInfo.Mode().Perm()
+	return perm&0100 != 0, nil
```

Comment:
I suggest move "permission patterns" to constants

E.g. `perm&isOwner`... `perm&isExecReadWrite` and so on.

Reply:
Fixed